sdk/provisioning/Azure.Provisioning.PostgreSql/src/PostgreSqlConnectionString.cs

```diff
@@ -19,6 +19,6 @@ internal PostgreSqlConnectionString(PostgreSqlFlexibleServer server, Parameter u
         {
         }
 
-        private static string GetParameterValue(Parameter parameter) => parameter.IsFromOutput ? parameter.Value! : parameter.Name;
+        private static string GetParameterValue(Parameter parameter) => parameter.IsFromOutput ? parameter.Value!.ToString()! : parameter.Name;
```

Comment:
Does this `ToString()` match bicep semantics for all values?

Reply:
ToString is not actually needed anymore.